Library/Tracking/KSRAnalytics.swift

```diff
@@ -733,14 +732,16 @@ public final class KSRAnalytics {
   public func trackAddOnsPageViewed(
     project: Project,
     reward: Reward,
-    refTag: RefTag?
+    refTag: RefTag?,
+    checkoutData: CheckoutPropertiesData
```

Comment:
I've noticed on all the other tracking methods, the `refTag` is the last parameter. Can you update this one as well?